.env hotfix (#58)
osu-tournament-rating/otr-docs: Applications/Web/Getting Started.md

Configure the `.env` file:
  
1. Rename `.env.example` to `.env`. It should now look like this:

   ```
OSU_CLIENT_ID=
AUTH_SECRET=
OTR_API_ROOT=
   ```

> 2. Copy your osu! API v2 client id and set `OSU_CLIENT_ID` equal to that value.

Reviewer: ```suggestion
2. Copy your osu! OAuth client ID and set `OSU_CLIENT_ID` equal to that value.
```

I think ID is technically the correct use when talking about identification

   > [!note] 

> > The API instance must also be using the same client id.

Reviewer: ```suggestion
> The API instance must also be using the same client ID.
```

> 
> This must be the same client used in your running instance of the API.

3. Set `AUTH_SECRET` to any string that is at least 32 characters in length.

>    > [!warning]

Reviewer: Is this supposed to be indented

Author: I indented the callouts so that they format as part of the list. Without the indent the callout begins at the leftmost margin. (see 1st image). With the indent they look like this. (2nd image). Let me know if you want these indented or not

![image](https://github.com/user-attachments/assets/525299c1-67ee-4e7b-b7cc-08210ebfc209)
![image](https://github.com/user-attachments/assets/d5ce8829-72bc-4dee-afbc-7a8cdae55f5a)

Reviewer: looks good indented

> This should be a secure string in production. This is the encryption key for session cookies.

5. Set `OTR_API_ROOT` to `http://localhost:5075`. The API runs on port `5075` by default.

> 

Reviewer: ```suggestion
```

Remove new line I think?
